openpype/pipeline/colorspace.py

```diff
@@ -601,3 +601,63 @@ def _get_imageio_settings(project_settings, host_name):
     imageio_host = project_settings.get(host_name, {}).get("imageio", {})
 
     return imageio_global, imageio_host
+
+
+def get_display_view_colorspace_name(config_path, display, view):
+    """Return colorspace name for the given display and view.
+
+    Args:
+        config_path (str): path string leading to config.ocio
+        display (str): display name e.g. "ACES"
+        view (str): view name e.g. "sRGB"
+
+    Returns:
+        view color space name (str) e.g. "Output - sRGB"
+    """
+
+    if not compatibility_check():
+        # python environment is not compatible with PyOpenColorIO
+        # needs to be run in subprocess
+        return get_display_view_colorspace_subprocess(config_path,
+                                                      display, view)
+
+    from openpype.scripts.ocio_wrapper import _get_display_view_colorspace_name  # noqa
+
+    return _get_display_view_colorspace_name(config_path, display, view)
+
+
+def get_display_view_colorspace_subprocess(config_path, display, view):
+    """get view colorspace name for the given display and view
+        via subprocess.
+
+    Args:
+        config_path (str): path string leading to config.ocio
+        display (str): display name e.g. "ACES"
+        view (str): view name e.g. "sRGB"
+
+    Returns:
+        view color space name (str) e.g. "Output - sRGB"
+    """
+
+    with _make_temp_json_file() as tmp_json_path:
+        # Prepare subprocess arguments
+        args = [
+            "run", get_ocio_config_script_path(),
+            "config", "get_display_view_colorspace_name",
+            "--in_path", config_path,
+            "--out_path", tmp_json_path,
+            "--display", display,
+            "--view", view
+
+        ]
+        log.info("Executing: {}".format(" ".join(args)))
```

Comment:
This should be debug log.
```suggestion
        log.debug("Executing: {}".format(" ".join(args)))
```